routing-java-ortools/VERSION

```diff
@@ -0,0 +1 @@
+v1.2.1
```

Comment:
Why are we starting this app at version `v1.2.1`? :thinking:

Reply:
Good catch. Set it to 1.0.0